.changeset/kind-beers-chew.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-api-docs': minor
```

Comment:
This shouldn't be breaking change for people using existing requestInterceptor ?

Reply:
requestInterceptor is defined in `swaggerUiProps`. So I don't think it is a breaking change 🤔